monix-reactive/shared/src/main/scala/monix/reactive/Consumer.scala

```diff
@@ -227,6 +227,17 @@ object Consumer {
   def raiseError[In, R](ex: Throwable): Consumer.Sync[In, R] =
     new RaiseErrorConsumer(ex)
 
+  /** A consumer that will return a collection of all the streamed
+   * values as a Scala `List`.
+   *
+   * Note that this method is functionally the same as the `.toListL` from [[Observable]]
+   *
+   * WARNING: For infinite streams the process will eventually
+   * blow up with an out of memory error.
+   */
+  def collect[A]: Consumer.Sync[A, List[A]] =
```

Comment:
`collect` usually uses partial function - what do you think about a name like `toList` (consistent with `toListL`) ?

Reply:
sure, btw, what does represent the final `L` from the `.toListL` ?

Reply:
amended all your comments, thank you for the quick feedback! :)